packages/patternfly-4/react-core/src/components/AboutModal/helpers/AboutModalBox.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/patternfly/components/AboutModalBox/about-modal-box.css';
+
+export interface AboutModalBoxProps extends React.HTMLProps<HTMLDivElement> {
+  /** content rendered inside the AboutModelBox.  */
+  children: React.ReactNode;
+  /** additional classes added to the AboutModalBox  */
+  className?: string;
+};
+
+export const AboutModalBox: React.SFC<AboutModalBoxProps> = ({
+  children,
+  className = '',
+  ...props
+}: AboutModalBoxProps) => (
+  <div {...props} role="dialog" aria-modal="true" className={css(styles.aboutModalBox, className)}>
```

Comment:
It probably does not make a difference since it is internal, but can you spread the props at the end here since you removed the aria attributes from the props so the overwrite any other props.